moveit_ros/moveit_servo/src/collision_check.cpp

```diff
@@ -218,7 +218,7 @@ void CollisionCheck::printCollisionPairs(collision_detection::CollisionResult::C
                                     << contact_map.begin()->first.first << ", " << contact_map.begin()->first.second);
     // Log all other contacts if in debug mode
     RCLCPP_DEBUG_STREAM_THROTTLE(LOGGER, clock, ROS_LOG_THROTTLE_PERIOD, "Objects in collision:");
-    for (auto contact : contact_map)
+    for (const auto& contact : contact_map)
```

Comment:
I imagine this change should also be made on ROS1? I can add it to the open PR I have over there where I have added other small changes found from the ROS2 review (https://github.com/ros-planning/moveit/pull/2249).

Reply:
[That line in ROS1](https://github.com/ros-planning/moveit/blob/ab68828e546ac6b9fdecb2c3d38fe07311a863d2/moveit_ros/moveit_servo/src/collision_check.cpp#L221)